sample-code/src/tutorials/mcp/weather-mcp-server.ts

```diff
@@ -3,18 +3,20 @@
 /* eslint-disable import/no-internal-modules */
 import { McpServer } from '@modelcontextprotocol/sdk/server/mcp.js';
 import { StdioServerTransport } from '@modelcontextprotocol/sdk/server/stdio.js';
-import * as z from 'zod/v3';
+import * as z from 'zod';
 
 const server = new McpServer({
   name: 'Open-Meteo Weather MCP Server',
   version: '1.0.0'
 });
 
-server.tool(
+server.registerTool(
```

Comment:
`tool()` deprecated, using `registerTool()` instead.

Reply:
As the line numbers changed a bit now, can you check if the referred line numbers in the documentation are still valid?